opm/parser/eclipse/EclipseState/Schedule/WellInjectionProperties.cpp

```diff
@@ -54,4 +55,20 @@ namespace Opm {
     bool WellInjectionProperties::operator!=(const WellInjectionProperties& other) const {
         return !(*this == other);
     }
+
+    std::ostream& operator<<( std::ostream& stream,
```

Comment:
Sort-of. It's useful for unit tests (makes BOOST_CHECK_EQUAL work) and possibly parser state output later. For BOOST_CHECK_EQUAL it's really only important that it *exists* whenever well properties are compared (e.g. the unit test file), but I figured it's just as nice to provide it generally. What it outputs really doesn't matter that much, but I have found them useful for debugging in the past.